demos/rag_agentic/notebooks/Level6_agents_MCP_and_RAG.ipynb

```diff
@@ -267,46 +225,95 @@
   },
   {
    "cell_type": "markdown",
-   "id": "2bf70ace-b704-4379-aa88-3c793cc4f959",
+   "id": "6bc8d3f8",
    "metadata": {},
    "source": [
-    "### System Prompts for different models\n",
+    "## Resolve and report any errors that might be happening on my OpenShift Cluster"
```

Comment:
I don't think we want to promise "resolving" problems here. 
```suggestion
    "## Discover and report solutions to errors that might be happening on my OpenShift Cluster"
```

Reply:
good point, will update